games/applications/__init__.py

```diff
@@ -0,0 +1,53 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the MIT License.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+
+"""Benchmarks of application circuits."""
+
+import pytest
+
+from qiskit.compiler import transpile
+from qiskit.providers.fake_provider import (
+    FakeWashington,
+    FakeBrooklyn,
+    FakeRochester,
+    FakeMontreal,
+    FakeCairo,
+    FakeToronto,
+    FakeGuadalupe,
+    FakeMelbourne,
+)
+from qiskit.quantum_info.analysis import hellinger_fidelity
+
+backends = [
+    FakeWashington(),
+    FakeBrooklyn(),
+    FakeRochester(),
+    FakeMontreal(),
+    FakeCairo(),
+    FakeToronto(),
+    FakeGuadalupe(),
+    FakeMelbourne(),
+]
+
+
+def run_qiskit_circuit(benchmark, circuit, backend, optimization_level, shots, expected_counts):
+    info, tqc = benchmark(
+        transpile,
+        circuit,
+        backend,
```

Comment:
Is it possible to have the framework report the architecture first, and then the backend name? e.g. `Eagle r1 (ibm_washington)` vs `Falcon r5.11 (ibm_cairo)` might help provide some additional context in comparing between the different backends.

Reply:
As long as we constrain this benchmark to targeting fake/ibm backends it's doable. I can change the string in `run_bv` to be something like: `f"{backend.configuration().processor_type['family']} r{backend.configuration().processor_type['revision']} ({backend.name()})`. This metadata is only really used by the ibm backends so it's not general if we wanted to target different backends here.